env.mjs

```diff
@@ -5,13 +5,10 @@ export const env = createEnv({
   server: {},
   client: {
     NEXT_PUBLIC_APP_URL: z.string().min(1),
-    NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID: z.string().min(1),
```

Comment:
I think we can't remove this lines

Reply:
<img width="1268" alt="Screenshot 2567-07-26 at 21 47 45" src="https://github.com/user-attachments/assets/e09a88a9-4e6d-453a-9387-6a40f4b8d882">

Reply:
add la thank you eieie